### PR TITLE
Update release notes for k8s 1.16 version

### PR DESCRIPTION
**Pull request by evrardjp**
For release 4.1.0

### CODE REVIEW

File: adoc/MAIN.release-notes.adoc [hunk 1]
```diff
@@ -15,7 +15,71 @@ The most recent version of the Release Notes is available online at https://www.
...
+Refer to: link:https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup[{sls} 15 SP1 Admin Guide: Updating Software with Zypper]
+
+==== Upgrade your {kube} cluster:
+
+. Check if upgrades are available:
+  ``skuba cluster upgrade plan``
+. Check upgrades at the node level
+  ``skuba node upgrade plan <nodename>``
+. Upgrade all control plane node, one by one, by running
+  ``skuba node upgrade apply`` on each of them.
```
Comment (davidcassany) [inline adoc/MAIN.release-notes.adoc]: Isn't it mandatory to upgrade master nodes first? I think so

**Review by davidcassany — COMMENTED** [review]
At glance I just have a comment about nodes upgrade order

File: adoc/MAIN.release-notes.adoc (hunk 1 above)
Comment (evrardjp) [reply adoc/MAIN.release-notes.adoc]: Yes. That's what was written as "control plane". To be consistent with other places I moved to a "master"/"worker" terminology in the last patchset.

**Comment by evrardjp** [comment]
Could you remove the label On-Hold please? It should be okay to review now.

**Comment by evrardjp** [comment]
Thanks for the update. Looks better indeed! :)

**Review by Itxaka — COMMENTED** [review]
lgtm +1

**Review by ereslibre — APPROVED** [review]
LGTM, thanks @evrardjp

**Pull request merged**